### DISCUSSION TITLE
マルチプレイのSDKで発行されたダウンロードライセンスの有効期間はどれくらいありますか？

### DISCUSSION THREAD

**Discussion by programus** [pixiv/vroid-sdk-developers#82] [Q&A] [answered]
README文書に以下の記述がありました。
> ダウンロードライセンスを発行するとき、同じユーザーが同じアプリケーションで発行した同じキャラクターの古いダウンロードライセンスは無効化されます。
ダウンロードライセンスの有効期限が切れているときや無効化されているときにMultiplayModelLoader.LoadVrmを使用するとonErrorが呼ばれ、エラーコードにはCOMMON_NOT_FOUNDが設定されます。

それに以下の記述もありました。
> ダウンロードライセンスはVRoid HubからVRMをダウンロードするために作られる有効期限付きのデータです。

発行されたダウンロードライセンスは同じユーザーが同じアプリケーションで発行し直さない限り、有効のままなのか、それとも、一定期間後自動的に無効になるか教えていただきたいです。後者だと思いますが、どれくらいの期間で無効になるかもご教示いただきたいです。

**Comment by nyamadan** [comment, marked as answer]
> 発行されたダウンロードライセンスは同じユーザーが同じアプリケーションで発行し直さない限り、有効のままなのか、それとも、一定期間後自動的に無効になるか教えていただきたいです。後者だと思いますが、どれくらいの期間で無効になるかもご教示いただきたいです。

後者となります。発行されたダウンロードライセンスは基本的に24時間で期限切れとなります。

**Reply by programus** [reply]
ご回答、ありがとうございました。
